notebook/spec2vec/tutorial.ipynb

```diff
@@ -63,73 +63,107 @@
+    "- `metadata`, metadata fields chosen with `include_metadata` in the input"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 3,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading GitHub repo omigami/romigami@MLOPS-306-ms2deepscore-refactor\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "   sh: error importing function definition for `kcron-activate'\n",
+      "   sh: error importing function definition for `kcron-suspend'\n",
+      "\u001b[32m✔\u001b[39m  \u001b[90mchecking for file ‘/private/var/folders/mn/vsnt67z141g88h5hlgl32gd40000gn/T/RtmpmIimSn/remotes26215fea2348/omigami-romigami-a91f53b/DESCRIPTION’\u001b[39m\u001b[36m\u001b[39m\n",
+      "\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mpreparing ‘romigami’:\u001b[39m\u001b[36m\u001b[39m\n",
+      "\u001b[32m✔\u001b[39m  \u001b[90mchecking DESCRIPTION meta-information\u001b[39m\u001b[36m\u001b[39m\n",
+      "\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mchecking for LF line-endings in source and make files and shell scripts\u001b[39m\u001b[36m\u001b[39m\n",
+      "\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mchecking for empty or unneeded directories\u001b[39m\u001b[36m\u001b[39m\n",
+      "\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mbuilding ‘romigami_1.0.tar.gz’\u001b[39m\u001b[36m\u001b[39m\n",
+      "   \n",
+      "\r"
+     ]
+    }
+   ],
    "source": [
     "# install devtools and romigami if not already installed\n",
-    "if(\"devtools\" %in% installed.packages() == FALSE) {install.packages(\"devtools\")}\n",
-    "devtools::install_github(\"omigami/romigami\")\n",
+    "#if(\"devtools\" %in% installed.packages() == FALSE) {install.packages(\"devtools\")}\n",
+    "devtools::install_github(\"omigami/romigami\", ref=\"MLOPS-306-ms2deepscore-refactor\", force=TRUE)\n",
     "\n",
     "# import the romigami package\n",
     "library(\"romigami\")"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "virtualenv: Romigami\n",
-      "Using virtual environment '/Users/user/.virtualenvs/Romigami' ...\n"
+      "Using virtual environment '/Users/admin/.virtualenvs/Romigami' ...\n"
      ]
     }
    ],
    "source": [
     "# environment setup\n",
-    "initialize_environment()"
+    "omigami <- omigami_init()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client <- omigami$Spec2Vec(token=\"73gyPBFLyA5zp1WYh0XzGBLMCiilJ3Gr\")"
```

Comment:
the token should be removed from the jupyter notebook and the R markdown files right ?

Reply:
good catch.